docs/web_reference.rst

```diff
@@ -702,6 +703,14 @@ StreamResponse
                           specification the cookie
                           conforms. (Optional, *version=1* by default)
 
+      .. versionadded:: 3.7
```

Comment:
Please move the directive inside `:param:` block, see `web.Response` `zlib_executor_size` for inspiration.